### PR TITLE
FEATURE: Add Process View

### LINKED ISSUE #51: Process view
```
http://127.0.0.1:8000/_VIEW/SCHEMA/VIEW?parameters...
```

The VIEW in PostgreSQL is a query processed, where we use as a table, see an example below:

```
SELECT field1, field2 FROM MY_VIEW;
```

*The VIEW must receive all parameters like SELECT (query)*

### PR DESCRIPTION
**Pull request by marioidival**
closes #51

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -60,6 +60,7 @@ func app() {
 	r.HandleFunc("/{database}/{schema}/{table}", controllers.InsertInTables).Methods("POST")
 	r.HandleFunc("/{database}/{schema}/{table}", controllers.DeleteFromTable).Methods("DELETE")
 	r.HandleFunc("/{database}/{schema}/{table}", controllers.UpdateTable).Methods("PUT", "PATCH")
+	r.HandleFunc("/{database}/{schema}/view/{view}", controllers.SelectFromViews).Methods("GET")
```
Comment (avelino) [inline cmd/root.go]: can I have a table named `view` that would be a problem, so I suggested we use `_view`

Comment (avelino) [inline cmd/root.go]: proposal:  `/_VIEW/{database}/{schema}/{view}`

`_VIEW` in uppercase

Comment (marioidival) [inline cmd/root.go]: @avelino got it

File: adapters/postgres/postgres.go [hunk 2]
```diff
@@ -600,3 +600,22 @@ func FieldsPermissions(table string, cols []string, op string) []string {
 	}
 	return nil
 }
+
```
Comment (felipeweb) [inline adapters/postgres/postgres.go]: fix lint warning here

**Pull request merged**